apps/authorization/routers.py

```diff
@@ -22,7 +22,7 @@
                                422: {"model": JSENDFailOutSchema, "description": "ValidationError"},
                                401: {"model": JSENDFailOutSchema, "description": "Not authorized response"}
                            },
-                           tags=["Authorization"])
+                           tags=["User dashboard"])
```

Comment:
why did u define "user dasboard", if the file is located in the authorization app?